Sources/KlaviyoSwift/KlaviyoState.swift

```diff
@@ -190,42 +191,53 @@ struct KlaviyoState: Equatable, Codable {
         var attributes = profile.data.attributes
         var location = profile.data.attributes.location ?? .init()
         var properties = profile.data.attributes.properties.value as? [String: Any] ?? [:]
+        let updatedProfile = Profile.updateProfileWithProperties(dict: pendingProfile)
 
-        // Optionally (if not already specified) overwrite attributes and location with pending profile information.
```

Comment:
This is the change I don't mind reverting as I don't think it improves the code. I made this change so that the logic to go over a pending profile dictionary and create a profile (that I use in creation of token request) can be reused.